templates/zerver/help/moderating-open-organizations.md

```diff
@@ -31,6 +31,8 @@ Zulip has many features designed to simplify moderation:
 
 * [Ban (deactivate) users](/help/deactivate-or-reactivate-a-user) acting in
   bad faith. You can reactivate them later if they repent.
+* [Mute users](/help/mute-a-user) to hide messages and block notifications
+  from them.
```

Comment:
I think this signals incorrectly that muting is a global response.  Maybe:

"Individual users can [mute users](/help/mute-a-user) to protect themselves until a moderator is able to respond to a problematic situation."